src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3341,7 +3341,11 @@ static void writeBreathMark(const Breath* const breath, XmlWriter& xml, Notation
         tagName += color2xml(breath);
         tagName += QString(" placement=\"%1\"").arg(breath->placement() == PlacementV::ABOVE ? "above" : "below");
 
-        xml.tagRaw(tagName, type);
+        if (breath->isCaesura()) { // for backwards compatibility, as 3.6.2 and earlier can't import those special caesuras,
+            xml.tagRaw(tagName);   // but reports corruption on all subsequent measures and imports them entirely empty.
```

Comment:
I'm not sure I like the idea of sacrificing the completeness of our MusicXML export to work around a bug in MuseScore 3...
We could create a setting for whether to do this or not. Something like "Optimize for use with MuseScore 3.6.2 and older". @oktophonie what do you think?

Reply:
No way.

Reply:
Seem my earlier comments on this, https://github.com/musescore/MuseScore/pull/20370#discussion_r1421763789

Reply:
Adding a setting for this